cli/cmd/agent.go

```diff
@@ -213,6 +295,53 @@ func init() {
 	agentInstallCmd.Flags().BoolVar(&agentCmdState.InstallTrustHostKey,
 		"trust_host_key", false, "automatically add host keys to the ~/.ssh/known_hosts file",
 	)
+
+	// 'agent ctf aws ec2ic' flags
+	agentCTFAWSEC2ICCmd.Flags().StringVar(&agentCmdState.CTFInfraTagKey,
+		"infra_tag_key", "", "only install agents on infra with this tag key set",
+	)
+	agentCTFAWSEC2ICCmd.Flags().StringArrayVar(&agentCmdState.CTFInfraTag,
+		"infra_tag", []string{}, "only install agents on infra with this tag",
+	)
+	agentCTFAWSEC2ICCmd.Flags().StringVar(&agentCmdState.InstallAgentToken,
+		"token", "", "agent access token",
+	)
+	agentCTFAWSEC2ICCmd.Flags().BoolVar(&agentCmdState.InstallTrustHostKey,
+		"trust_host_key", false, "automatically add host keys to the ~/.ssh/known_hosts file",
+	)
+	agentCTFAWSEC2ICCmd.Flags().StringArrayVarP(&agentCmdState.CTFIncludeRegions,
+		"include_regions", "r", []string{}, "list of regions to filter on",
```

Comment:
Will there be an `exclude_regions` flag in the future? or why prefix this with `include_`?

If the answer is yes, leave it, if no, I rather drop the prefix.
```suggestion
		"regions", "r", []string{}, "list of regions to filter on",
```

Reply:
There will be an `exclude_regions` flag, tracked at https://lacework.atlassian.net/browse/RAIN-37868. Not sure if I'll get it in this PR though